kafka/channel/pkg/reconciler/controller/kafkachannel.go

```diff
@@ -74,10 +74,6 @@ const (
 	dispatcherName = "kafka-ch-dispatcher"
 )
 
-func newReconciledNormal(namespace, name string) pkgreconciler.Event {
-	return pkgreconciler.NewEvent(corev1.EventTypeNormal, "KafkaChannelReconciled", "KafkaChannel reconciled: \"%s/%s\"", namespace, name)
-}
```

Comment:
The reason for this is explained here: https://github.com/knative/pkg/issues/1520